CMakeLists.txt

```diff
@@ -192,6 +196,30 @@ endif()
 # Build Tests and download Catch2
 if (${PARTHENON_ENABLE_UNIT_TESTS} OR ${PARTHENON_ENABLE_INTEGRATION_TESTS} OR ${PARTHENON_ENABLE_REGRESSION_TESTS} OR ${PARTHENON_ENABLE_PERFORMANCE_TESTS})
 
+  # if running regression test automatically synch gold standard
+  if (${PARTHENON_ENABLE_REGRESSION_TESTS} AND ${REGRESSION_GOLD_STANDARD_SYNC})
+    if (EXISTS ${PROJECT_SOURCE_DIR}/tst/regression/gold_standard/current_version)
+      file(READ ${PROJECT_SOURCE_DIR}/tst/regression/gold_standard/current_version REGRESSION_GOLD_STANDARD_ON_DISK)
+    else()
+      set(REGRESSION_GOLD_STANDARD_ON_DISK 0)
+    endif()
+    if (NOT (${REGRESSION_GOLD_STANDARD_ON_DISK} EQUAL ${REGRESSION_GOLD_STANDARD}))
+      ExternalProject_Add(
+        gold_standard
+        URL https://github.com/lanl/parthenon/releases/download/regression-gold-v${REGRESSION_GOLD_STANDARD}/parthenon_regression_gold_v${REGRESSION_GOLD_STANDARD}.tgz
+        URL_HASH SHA512=45f57d16b76a3a44940e40ea642c1d5a2b3aea681a3064e4c06b031bf264d627f8009e967b92a2cb0fbc08e5cd0ffda381038483f390fa00f47c42a551ca3646
```

Comment:
Can we make these variable set further up in the script? Or even better, can we auto-compute the hash? When the version changes, presumably the hash will too... I can see this being annoying to maintain.

Reply:
The hash is computed manually for the file downloaded and then compared against this reference value. This way it's ensured that the file downloaded is not corrupted or altered.
Given that the hash and the version change at the same time (and likely very infrequently) I don't see this as too annoying to maintain. However, if this is an issue we could remove potentially remove the hash.
Anyway, now moved the HASH variable further up next to the version as those two would be updated simultaneously.

Reply:
I think it's very likely that in debugging, the file will be updated frequently. Since a user may want to run tests against an older version or against their own file. So my preference would be to remove the `HASH`. But it's not a big deal.

Reply:
oh I see. The hash only matters for downloading the file, not locally. Never mind. This is fine with me.